css/src/admin-global.scss

```diff
@@ -452,8 +448,6 @@ td.column-wpseo-linked {
 	background: transparent;
 	box-shadow: none;
 	vertical-align: top;
-	/* IE 11 */
-	cursor: pointer;
```

Comment:
Looks like this comment was originally meant for `overflow: visible` see https://github.com/Yoast/wordpress-seo/blame/40b3f1c423bc5e69fc0e83be8eca791ac99d9fac/css/metabox-310.css#L356
Then, a merge moved the comment. `cursor: pointer;` should be kept because it's needed for all browsers. `overflow: visible;` should be removed instead.